File: 1946A. Median of an Array.py
def solve():
    n = int(input())
    arr = list(map(int, input().split()))

    arr.sort()

    med = (n + 1) // 2
    ans = 0
    for i in range(med, n + 1):
        if arr[i - 1] == arr[med - 1]:
            ans += 1

    print(ans)

def main():
    tc = int(input())

    for _ in range(tc):
        solve()

if __name__ == "__main__":
    main()

File: 1946A. Median of an Array.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int tc = scanner.nextInt();
        scanner.nextLine();

        while (tc-- > 0) {
            solve(scanner);
        }

        scanner.close();
    }

    public static void solve(Scanner scanner) {
        int n = scanner.nextInt();
        int[] arr = new int[n + 1];

        for (int i = 1; i <= n; i++) {
            arr[i] = scanner.nextInt();
        }

        Arrays.sort(arr, 1, n + 1);

        int med = (n + 1) / 2;
        int ans = 0;
        for (int i = med; i <= n; i++) {
            if (arr[i] == arr[med]) {
                ans++;
            }
        }

        System.out.println(ans);
    }
}
